### ISSUE TITLE
UI for Core spell-check

### ISSUE THREAD

**Issue by quicoto** [jesseweed/seti-ui#256]
Hello,

I'm running **Atom 1.7.3** and **seti-ui 1.0.4**

When using atom's core spell-check the UI for the dropdown options has some issues.

**Is this something that can be fixed?**

Thank you very much

![seti-ui spell-chek](https://cldup.com/q2Ez0q_uqa.gif)

**Comment by quicoto** [comment]
I think this issue might be the same as: https://github.com/jesseweed/seti-ui/issues/242

**Comment by bruinebeer** [comment]
Yep it is. This doesn't fix it 100%. But it should help you further until it gets fixed.

```Less
atom-text-editor {
  atom-overlay {
    .corrections {
      border: 4px solid black;
      border-radius: 3px;
      border-top: 8px solid black;
    }
    .list-group{
      background-color: black;
    }
    .corrections .list-group .selected {
      background-color: #222;
    }
    .corrections .list-group .selected::before,
    .corrections .list-tree .selected::before {
      background-color: rgba(0,0,0,0);
    }
  }
}
```

You should play around with the colours a bit to match your liking.

**Comment by jesseweed** [comment]
Ah! Spell check in one I forgot to check! I'll take a look at this and get a fix pushed out in the next update.

**Comment by jesseweed** [comment]
This should be fixed in 1.2.1 update. Feel free to re-open is you still have an issue.
